src/brokers/add-broker/AddBroker.container.tsx

```diff
@@ -20,7 +21,7 @@ const AddBrokerPage: FC<AddBrokerPageProps> = ({ match }) => {
     apiVersion: 'broker.amq.io/v1beta1',
     kind: 'ActiveMQArtemis',
     metadata: {
-      name: 'default',
+      name: '',
```

Comment:
```suggestion
      name: 'default',
```

Let's keep the default value of  name field.